puppet/modules/slave/manifests/packaging/rpm.pp

```diff
@@ -33,6 +33,14 @@
   $yaml = if $facts['os']['release']['major'] == '7' { 'python36-PyYAML' } else { 'python3-pyyaml' }
   ensure_packages(['python3', $yaml])
 
+  # Required by Obsah / Obal
+  if $is_el8 {
+    package { 'pkg_resources':
```

Comment:
This is part of setuptools. Can you install `python3-setuptools` or `python3.11-setuptools`? instead? Given we install `python3-pyyaml` above I'd think `python3-setuptools` is the right package.

Reply:
Ahh yes, that did it! Thanks!